<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMOAnakartListeFormu</class>
 <widget class="QDialog" name="TMOAnakartListeFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
TMOAnakartListeFormu{

	background-image: url(:/resimler/anakart.jpg);
}

QLabel {
   font:Bold; 
   color: #ffffff;
}
QGroupBox {
    font:Bold;
color: #ffffff;
}
QPushButton{font:Bold;}
QRadioButton{
    font:Bold;
color: #ffffff;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QGroupBox" name="GPAramaBolumu">
     <property name="title">
      <string> ANAKART ARAMA BÖLÜMÜ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="GPParcaMarkasi">
        <property name="title">
         <string>Parça Markası</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="LPMarkasiDeger">
           <property name="text">
            <string>Değer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LEPMarkasiDeger"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="GPPMarkasiFiltreTuru">
           <property name="title">
            <string>Filtre Türü</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QRadioButton" name="RBPMarkasiFTIleBaslayan">
              <property name="text">
               <string>İle Başlayan</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="RBPMarkasiFTIleBiten">
              <property name="text">
               <string>İle Biten</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="RBPMarkasiFTIceren">
              <property name="text">
               <string>İçeren</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="GPParcaModeli">
        <property name="title">
         <string>Parça Modeli</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="LPModeliDeger">
           <property name="text">
            <string>Değer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LEPModeliDeger"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="GPPModeliFiltreTuru">
           <property name="title">
            <string>Filtre Türü</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QRadioButton" name="RBPModeliFTIleBaslayan">
              <property name="text">
               <string>İle Başlayan</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="RBPModeliIFTIleBiten">
              <property name="text">
               <string>İle Biten</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="RBPModeliFTIceren">
              <property name="text">
               <string>İçeren</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="PBAra">
        <property name="text">
         <string>ARA</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="PBKapat">
        <property name="text">
         <string>KAPAT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="TWTablo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PBAra</sender>
   <signal>clicked()</signal>
   <receiver>TMOAnakartListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1079</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>1103</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PBKapat</sender>
   <signal>clicked()</signal>
   <receiver>TMOAnakartListeFormu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1147</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>1191</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TMOAnakartListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TMOAnakartListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIceren</sender>
   <signal>clicked()</signal>
   <receiver>TMOAnakartListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliFTIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TMOAnakartListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>1204</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliIFTIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TMOAnakartListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>728</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>1203</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliFTIceren</sender>
   <signal>clicked()</signal>
   <receiver>TMOAnakartListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>902</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>1203</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LEPMarkasiDeger</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TMOAnakartListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>-2</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LEPModeliDeger</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TMOAnakartListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aramaYap()</slot>
 </slots>
</ui>
